public/app/features/dashboard-scene/settings/GeneralSettingsEditView.tsx

```diff
@@ -271,6 +276,24 @@ export class GeneralSettingsEditView
             >
               <RadioButtonGroup onChange={model.onTooltipChange} options={GRAPH_TOOLTIP_OPTIONS} value={graphTooltip} />
             </Field>
+
+            <Field
+              label="Preload panels"
+              description="When enabled all panels will start loading as soon as the dashboard has been loaded"
```

Comment:
we could mention there may be a performance impact to this 🤔

Reply:
we should do the translations here,

basically as the example below then run `make i18n-extract`